internal/consensus/reactor.go

```diff
@@ -82,8 +82,6 @@ const (
 	listenerIDConsensus = "consensus-reactor"
 )
 
-type ReactorOption func(*Reactor)
-
 // NOTE: Temporary interface for switching to block sync, we should get rid of v0.
 // See: https://github.com/tendermint/tendermint/issues/4595
 type BlockSyncReactor interface {
```

Comment:
You could also delete this while you're at it :)

Reply:
someday! this actualy looks like it gets used :(